frontend/src/lib/apiClient.js

```diff
@@ -76,6 +76,12 @@ const resetPassword = (resetPasswordToken, password) => apiClient.put('/api/v1/p
   },
 });
 
+const resetLoginPassword = password => apiClient.put('/api/v1/password', {
```

Comment:
I'd be inclined to name this something like `resetCurrentUserPassword` and potentially rename the other function to `resetPasswordByToken`.